src/routing/future.rs

```diff
@@ -179,3 +181,9 @@ where
         self.project().inner.poll(cx)
     }
 }
+
+opaque_future! {
+    /// Response future from [`MakeRouteService`] services.
+    pub type MakeRouteServiceFuture<S> =
+        futures_util::future::Ready<Result<S, Infallible>>;
```

Comment:
With axum's current MSRV you could use the `std` equivalent (introduced in 1.48.0). Or does that not work due to dependencies using this type?

Reply:
Whoa I did not realize `std::future::Ready` was stable 🎊 Gonna try using that instead!

Reply:
I'll address this in a follow up PR.